infra/test/cardano-sl-infra-test.cabal

```diff
@@ -0,0 +1,33 @@
+name:                cardano-sl-infra-test
+version:             1.1.0
```

Comment:
I think that should actual be `1.3.0` to match the other packages.

Reply:
Do you mean `1.2.0`? Just clarifying.

Reply:
Sorry, @Jimbo4350 just pointed me to https://github.com/input-output-hk/cardano-sl/pull/3064

Reply:
So we're indeed going with `1.3.0`.